ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -245,21 +248,23 @@ public List<String> checkSuspending() {
     });
 
     List<TaskRuntime> suspendingTasks = executingTaskMap.values().stream()
+        .filter(task -> task.getStatus().equals(TaskRuntime.Status.SCHEDULED) ||
+            task.getStatus().equals(TaskRuntime.Status.ACKED))
         .filter(task -> task.isSuspending(currentTime, taskAckTimeout) ||
             expiredOptimizers.contains(task.getOptimizingThread().getToken()) ||
             !authOptimizers.containsKey(task.getOptimizingThread().getToken()))
         .collect(Collectors.toList());
     suspendingTasks.forEach(task -> {
       LOG.info("Task {} is suspending, since it's optimizer is expired, put it to retry queue, optimizer {}",
-          task.getTaskId(), task.getOptimizingThread());
+              task.getTaskId(), task.getOptimizingThread());
```

Comment:
Can you rollback change here, the project use 4 space for continuation indent.

Reply:
ok